aspnetcore/client-side/using-browserlink.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Using Browser Link | Microsoft Docs
+author: ncarandini
+description: A Visual Studio feature that links the development environment with one or more web browsers
+keywords: ASP.NET Core, browser link, CSS sync
+ms.author: riande
+manager: wpickett
+ms.date: 12/28/2016
+ms.topic: article
+ms.assetid: 11813d4c-3f8a-445a-b23b-e4a57d001abc
+ms.technology: aspnet
+ms.prod: aspnet-core
+uid: client-side/using-browserlink
+---
+# Using Browser Link
+
+By [Nicolò Carandini](https://github.com/ncarandini), [Mike Wasson](https://github.com/MikeWasson), and [Tom Dykstra](https://github.com/tdykstra)
+
+Browser Link is a feature in Visual Studio that creates a communication channel between the development environment and one or more web browsers. You can use Browser Link to refresh your web application in several browsers at once, which is useful for cross-browser testing.
+
+## Browser Link setup
+
+The ASP.NET Core **Web Application** project templates in Visual Studio 2015 and later for include everything needed for Browser Link.
+
+To add Browser Link to a project that you created by using the ASP.NET Core **Empty** or **Web API** template, follow these steps:
+
+1. Add the *Microsoft.VisualStudio.Web.BrowserLink.Loader* package 
+2. Add configuration code in the *Startup.cs* file.
+
+### Add the package
+
+Since this is a Visual Studio feature, the easiest way to add the package is to open the **Package Manager Console** (**View > Other Windows > Package Manager Console**) and run the following command:
+
+```console
+install-package Microsoft.VisualStudio.Web.BrowserLink.Loader
+```
+
+Alternatively, you can use **NuGet Package Manager**.  Right-click the project name in **Solution Explorer**, and choose **Manage NuGet Packages**. 
+
+![Open NuGet Package Manager](using-browserlink/_static/open-nuget-package-manager.png)
+
+Then find and install the package.
+
+![Add package with NuGet Package Manager](using-browserlink/_static/add-package-with-nuget-package-manager.png)
+
+### Add configuration code
+
+Open the *Startup.cs* file, and in the `Configure` method add the following code:
+
+```csharp
+app.UseBrowserLink();
+```
+
+Usually that code is inside an `if` block that enables Browser Link only in the Development environment, as shown here:
+
+![UseBrowserLink code](using-browserlink/_static/use-browserlink-code.png)
```

Comment:
We prefer not to use images for code.  Ideally this should be a snippet that refers to a file in a sample project located in the "sample" folder.